.github/workflows/continuous-integration.yml

```
@@ -96,50 +96,106 @@ jobs:
+            exit 1
+          fi
       - name: "PHP CodeSniffer"
-        if: ${{ !cancelled() && hashFiles(format('{0}/vendor/bin/phpcs', inputs.plugin-key)) != '' && hashFiles(format('{0}/.phpcs.xml', inputs.plugin-key)) != '' }}
+        if: ${{ !cancelled() && hashFiles(format('{0}/.phpcs.xml', inputs.plugin-key)) != '' }}
         run: |
           echo -e "\033[0;33mExecuting PHP CodeSniffer...\033[0m"
+          if [[ -f "vendor/bin/phpcs" ]]; then
+            vendor/bin/phpcs
+          elif [[ -f "../../vendor/bin/phpcs" ]]; then
+            ../../vendor/bin/phpcs
+          else
+            echo -e "\033[0;31mPHP CodeSniffer binary not found!\033[0m"
+            exit 1
+          fi
           vendor/bin/phpcs
       - name: "PHP-CS-Fixer"
-        if: ${{ !cancelled() && hashFiles(format('{0}/vendor/bin/php-cs-fixer', inputs.plugin-key)) != '' && hashFiles(format('{0}/.php-cs-fixer.php', inputs.plugin-key)) != '' }}
+        if: ${{ !cancelled() && hashFiles(format('{0}/.php-cs-fixer.php', inputs.plugin-key)) != '' }}
         run: |
           echo -e "\033[0;33mExecuting PHP-CS-Fixer...\033[0m"
-          PHP_CS_FIXER_IGNORE_ENV=1 vendor/bin/php-cs-fixer check --config=.php-cs-fixer.php --no-interaction --show-progress none
+          if [[ -f "vendor/bin/php-cs-fixer" ]]; then
+            PHP_CS_FIXER_IGNORE_ENV=1 vendor/bin/php-cs-fixer check --config=.php-cs-fixer.php --no-interaction --show-progress none
+          elif [[ -f "../../vendor/bin/php-cs-fixer" ]]; then
+            PHP_CS_FIXER_IGNORE_ENV=1 php ../../vendor/bin/php-cs-fixer check --config=.php-cs-fixer.php --no-interaction --show-progress none
+          else
+            echo -e "\033[0;31mPHP-CS-Fixer binary not found!\033[0m"
+            exit 1
+          fi
       - name: "PHPStan"
-        if: ${{ !cancelled() && hashFiles(format('{0}/vendor/bin/phpstan', inputs.plugin-key)) != '' && hashFiles(format('{0}/phpstan.neon', inputs.plugin-key)) != '' }}
+        if: ${{ !cancelled() && hashFiles(format('{0}/phpstan.neon', inputs.plugin-key)) != '' }}
         run: |
           echo -e "\033[0;33mExecuting PHPStan...\033[0m"
-          vendor/bin/phpstan analyze --ansi --memory-limit=2G --no-interaction --no-progress
+          if [[ -f "vendor/bin/phpstan" ]]; then
+            vendor/bin/phpstan analyze --ansi --memory-limit=2G --no-interaction --no-progress
+          elif [[ -f "../../vendor/bin/phpstan" ]]; then
+            ../../vendor/bin/phpstan analyze --ansi --memory-limit=2G --no-interaction --no-progress
+          else
+            echo -e "\033[0;31mPHPStan binary not found!\033[0m"
+            exit 1
+          fi
       - name: "Psalm"
         if: ${{ !cancelled() && hashFiles(format('{0}/psalm.xml', inputs.plugin-key)) != '' }}
         run: |
           echo -e "\033[0;33mExecuting Psalm...\033[0m"
           php ../../vendor/bin/psalm --long-progress --output-format=github
```

We should add the same logic for this one.

```suggestion
          if [[ -f "vendor/bin/psalm" ]]; then
            vendor/bin/psalm --long-progress --output-format=github
          elif [[ -f "../../vendor/bin/psalm" ]]; then
            ../../vendor/bin/psalm --long-progress --output-format=github
          else
            echo -e "\033[0;31mPsalm binary not found!\033[0m"
            exit 1
          fi
```